tools/ramdump/Ramdump_windows.ps1

```diff
@@ -0,0 +1,238 @@
+# ------------------------------  PARAMETERS ---------------------------------
+#$COM = [System.IO.Ports.SerialPort]::getportnames()
+$COM = 'COM12'

```

Comment:
How about asking user for the com port instead of hard coding??

Reply:
Modified the code, please check.